src/App/Markdown.php

```diff
@@ -21,4 +21,29 @@
 
 class Markdown
 {
+	// Mock test
+    public function __construct($argument1)
+    {
+        // TODO: write logic here
+    }
+
+    public static function createForWriting($argument1)
+    {
+        $markdown = new Markdown($argument1);
```

Comment:
`$markdown = new static($argument1);` as no indication the class won't be extended.